webservices/common/models/reports.py

```diff
@@ -25,91 +50,91 @@ class CommitteeReports(PdfMixin, BaseModel):
     __tablename__ = 'ofec_reports_house_senate_mv'
 
-    aggregate_amount_personal_contributions_general = db.Column(db.Numeric(30, 2))
-    aggregate_contributions_personal_funds_primary = db.Column(db.Numeric(30, 2))
-    all_other_loans_period = db.Column(db.Numeric(30, 2))
-    all_other_loans_ytd = db.Column(db.Numeric(30, 2))
-    candidate_contribution_period = db.Column(db.Numeric(30, 2))
-    candidate_contribution_ytd = db.Column(db.Numeric(30, 2))
-    gross_receipt_authorized_committee_general = db.Column(db.Numeric(30, 2))
-    gross_receipt_authorized_committee_primary = db.Column(db.Numeric(30, 2))
-    gross_receipt_minus_personal_contribution_general = db.Column(db.Numeric(30, 2))
-    gross_receipt_minus_personal_contributions_primary = db.Column(db.Numeric(30, 2))
-    loan_repayments_candidate_loans_period = db.Column(db.Numeric(30, 2))
-    loan_repayments_candidate_loans_ytd = db.Column(db.Numeric(30, 2))
-    loan_repayments_other_loans_period = db.Column(db.Numeric(30, 2))
-    loan_repayments_other_loans_ytd = db.Column(db.Numeric(30, 2))
-    loans_made_by_candidate_period = db.Column(db.Numeric(30, 2))
-    loans_made_by_candidate_ytd = db.Column(db.Numeric(30, 2))
-    net_contributions_ytd = db.Column(db.Numeric(30, 2))
-    net_operating_expenditures_ytd = db.Column(db.Numeric(30, 2))
-    net_contributions_period = db.Column(db.Numeric(30, 2), index=True)
-    net_operating_expenditures_period = db.Column(db.Numeric(30, 2))
-    operating_expenditures_period = db.Column(db.Numeric(30, 2))
-    operating_expenditures_ytd = db.Column(db.Numeric(30, 2))
-    other_receipts_period = db.Column(db.Numeric(30, 2))
-    other_receipts_ytd = db.Column(db.Numeric(30, 2))
-    refunds_total_contributions_col_total_ytd = db.Column(db.Numeric(30, 2))
-    subtotal_period = db.Column(db.Numeric(30, 2))
-    total_contribution_refunds_col_total_period = db.Column(db.Numeric(30, 2))
-    total_contributions_column_total_period = db.Column(db.Numeric(30, 2))
-    total_loan_repayments_made_period = db.Column(db.Numeric(30, 2))
-    total_loan_repayments_made_ytd = db.Column(db.Numeric(30, 2))
-    total_loans_received_period = db.Column(db.Numeric(30, 2))
-    total_loans_received_ytd = db.Column(db.Numeric(30, 2))
-    total_offsets_to_operating_expenditures_period = db.Column(db.Numeric(30, 2))
-    total_offsets_to_operating_expenditures_ytd = db.Column(db.Numeric(30, 2))
-    total_operating_expenditures_period = db.Column(db.Numeric(30, 2))
-    total_operating_expenditures_ytd = db.Column(db.Numeric(30, 2))
-    transfers_from_other_authorized_committee_period = db.Column(db.Numeric(30, 2))
-    transfers_from_other_authorized_committee_ytd = db.Column(db.Numeric(30, 2))
-    transfers_to_other_authorized_committee_period = db.Column(db.Numeric(30, 2))
-    transfers_to_other_authorized_committee_ytd = db.Column(db.Numeric(30, 2))
+    aggregate_amount_personal_contributions_general = db.Column(db.Numeric(30, 2))#missing
```

Comment:
I think this is pre-millionaire's amendment field. We don't need it going forward
http://www.fec.gov/press/bkgnd/MillionairesAmendment.html